cmd/argocd-server/commands/argocd_server.go

```diff
@@ -184,6 +185,7 @@ func NewCommand() *cobra.Command {
 				RedisClient:           redisClient,
 				StaticAssetsDir:       staticAssetsDir,
 				ApplicationNamespaces: applicationNamespaces,
+				EnableProxyExtension:  enableProxyExtension,
```

Comment:
Can we hard-code this to false until the RBAC enforcement PR is ready? People are silly and use development images. I'd rather avoid letting them shoot themselves in the foot, enabling a not-yet-complete feature.

Reply:
This is what this line is doing:

```golang
command.Flags().BoolVar(&enableProxyExtension, "enable-proxy-extension", env.ParseBoolFromEnv("ARGOCD_SERVER_ENABLE_PROXY_EXTENSION", false), "Enable Proxy Extension feature")
```

The second argument is the hardcoded default value:

```golang
func ParseBoolFromEnv(envVar string, defaultValue bool) bool
```

Reply:
Right, I understand that the _default_ value is hard-coded. I'd like to make it impossible for anyone to ever set anything besides `false` here, until auth work is done in the follow-up PR.

Basically I'd like to make this dead code.

Reply:
dead code now